Updated URL for Configuring Alternate Login ID

YuriySamorodov opened the pull request:
Previous link was pointing to outdated article on Technet.
Updated it with docs.microsoft.com address.
I guess it is good to update this URL for all languages, not only English.

azurecla commented:
Hi __@jastru__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

YuriySamorodov committed:
Updated URL for Configuring Alternate Login ID

YuriySamorodov committed:
Removed hardcoded remove the hardcoded "/en-us"

Removed the hardcoded language "/en-us" in the URL per @MicrosoftGuyJFlo request
